services/rfq/api/rest/versions.go

```diff
@@ -0,0 +1,13 @@
+package rest
+
+var versionsJSON = `{
```

Comment:
This is a nit but IMO it would be cleaner to have a go struct defined here instead of parsing json

Something like:

```go
type version struct {
  version string
  date time.Time
  comment string
  alerts string
}

var versions = []version{
{
  Version: "1.0",
  Date: time.MustParse("2024-01-01"),
}
}
```

If we think we want to parse from json in the future can always add `json:` struct tags

Reply:
agree - will change